docs/reference/repositories-stats/apis/clear-repositories-stats-archive.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[clear-repositories-stats-archive-api]]
```

Comment:
We can state that this API is not stable and can suffer changes, I think it can be useful for consumers of this API to have the information at hand. But I'm not sure what's our policy here.

Reply:
For now, we can just put on the "experimental tag", and mark the license as basic.

Reply:
We can also add a header that this is an API meant to be used by Elastic Cloud's service, to avoid any confusion.